packages/test/src/helpers-integration.ts

```diff
@@ -101,5 +108,31 @@ export function helpers(t: ExecutionContext<Context>): Helpers {
         ...opts,
       });
     },
+    async assertWorkflowUpdateFailed(
+      p: Promise<any>,
+      errorConstructor: ErrorConstructor,
```

Comment:
`errorConstructor` => `causeConstructor`

Reply:
nit: Might also consider moving `causeConstructor` and `message` to some `expectations` object, to make this helper more reusable in various contexts...